artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.EditTextSuggestions/Infrastructure/ITaskSuggestionService.cs

```diff
@@ -4,5 +4,6 @@ namespace Artificial.Scrum.Master.EditTextSuggestions.Infrastructure;
 
 public interface ITaskSuggestionService
 {
-    Task<GetEditTaskSuggestionResult?> GetEditTaskSuggestion(string userStoryTitle,string taskTitle, string taskDescription);
+    Task<GetEditTaskSuggestionResult?> GetEditTaskSuggestion(string userStoryTitle, string taskTitle,
+        string taskDescription);
```

Comment:
jak już to formatujesz to zrób to dobrze:
```suggestion
Task<GetEditTaskSuggestionResult?> GetEditTaskSuggestion(
    string userStoryTitle,
    string taskTitle,
    string taskDescription);
```